examples/demo/basic/scatter_rect_select.py

```diff
@@ -96,7 +96,6 @@ def _create_plot_component():
 # Attributes to use for the plot view.
 size = (650, 650)
 title = "Scatter plot with selection"
-bg_color = "lightgray"
```

Comment:
why not pass this to the `ComponentEditor` - like what we do in the other examples

Reply:
I can add that here in this PR.  I deleted it just because previously this `bg_color` variable was just unused entirely.

It makes sense to actually use it though